corehq/apps/reports/standard/cases/utils.py

```diff
@@ -171,3 +177,12 @@ def _get_location_accessible_ids(request):
 def query_location_restricted_cases(query, request):
     accessible_ids = _get_location_accessible_ids(request)
     return query.OR(case_es.owner(accessible_ids))
+
+
+def get_most_recent_case_type(domain):
+    # gets most recent case type
+    query = (CaseSearchES().domain(domain)
+             .NOT(case_es.case_type(USER_LOCATION_OWNER_MAP_TYPE)))
```

Comment:
should this be a default filter on the CaseSearch index? I wonder if anyone would ever want to search these cases?

Reply:
can this kind of change be part of a different PR?